src/views/ChartPage.vue

```diff
@@ -0,0 +1,463 @@
+<template>
+  <div :class="this.$store.state.css">
+    <div class="chartPage menuLayout row col-12 offset-0 col-sm-12 offset-sm-0 col-md-10 offset-md-1 col-lg-12 offset-lg-0 col-xl-10 offset-xl-1 col-xxl-8 offset-xxl-2">
+      <div class="header fixed title">
+        <select v-model="currentGraph" class="selectGraph">
+          <option value="pie">{{ $t('PIE_CHART') }}</option>
+          <option value="bar">{{ $t('BAR_CHART') }}</option>
+        </select>
+      </div>
+      <div class="placeholder top">
+        <select v-model="currentGraph" class="selectGraph">
+          <option value="pie">{{ $t('PIE_CHART') }}</option>
+          <option value="bar">{{ $t('BAR_CHART') }}</option>
+        </select>
+      </div>
+
+      <div class="row dateNav mobileVersion">
```

Comment:
Could this be extracted as a component ?

Reply:
done